.travis.yml

```diff
@@ -4,9 +4,25 @@ php:
   - 5.4
   - 5.5
 
+env:
+  - SYMFONY_VERSION="2.3.*"
+  - SYMFONY_VERSION="2.6.*"
+  - SYMFONY_VERSION="2.7.*"
+  - SYMFONY_VERSION="2.8.x-dev"
+  - SYMFONY_VERSION="3.0.x-dev"
+
+matrix:
+  allow_failures:
+    - env: SYMFONY_VERSION="2.8.x-dev"
+    - env: SYMFONY_VERSION="3.0.x-dev"
+
+
 before_script:
   - curl --silent http://getcomposer.org/installer | php
-  - php composer.phar install --dev --no-interaction --prefer-source
+  - >
+      php composer.phar require --dev \
+        symfony/symfony:${SYMFONY_VERSION}
```

Comment:
All your Travis builds are requiring the full Symfony package with a specific version. This means that nothing ensures that a normal composer install (i.e. without altering the composer deps first) would allow to have a working setup.

See https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/.travis.yml for an example doing this only for some jobs